src/components/Tasks/AddNewTaskItem.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useContext } from 'react';
+
+import ICAL from 'ical.js';
+import uuid from 'uuid';
+
+import Checkbox from '@material-ui/core/Checkbox';
+import TextField from '@material-ui/core/TextField';
+
+import { TaskType, TaskStatusType, TaskPriorityType } from '../../pim-types';
+import { ListItem } from '../../widgets/List';
+import { PimContext } from '../../Pim';
+
+const AddNewTaskItem = () => {
+  const [title, setTitle] = useState('');
+  const { onItemSave: save, collectionsTaskList: collections } = useContext(PimContext);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(e.target.value);
+  };
+
+  const handleKeyPress = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.key === 'Enter') {
+      e.preventDefault();
+
+      const event = new TaskType(null);
```

Comment:
A bit of a nit-pick, but looks like a bad copy-paste. The variable should be called "task" as it's a task. :)